plugins/poll/assets/javascripts/discourse/initializers/add-poll-ui-builder.js

```diff
@@ -17,10 +17,7 @@ function initializePollUIBuilder(api) {
       return (
         siteSettings.poll_enabled &&
         (composer.model.topic?.pm_with_non_human_user ||
-          (currentUser &&
-            (currentUser.staff ||
-              currentUser.trust_level >=
-                siteSettings.poll_minimum_trust_level_to_create)))
+          (currentUser && (currentUser.staff || currentUser.can_create_poll)))
```

Comment:
![](https://media2.giphy.com/media/bhHdL7eQuA85sCgHAX/giphy.gif)

Reply:
I still have to write this up internally, but we will need to do this in more places too because of the "not all groups are serialized to client" debacle